home/.bin/create-feature

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env ruby
+require_relative 'feature'
+
+help_text = <<-eoht
```

Comment:
convention here is that if there are any flags you can enumerate them here, but this is grood enough